lang/golang/golang/Makefile

```diff
@@ -13,8 +13,13 @@ PKG_VERSION:=$(GO_VERSION_MAJOR_MINOR)$(if $(GO_VERSION_PATCH),.$(GO_VERSION_PAT
 PKG_RELEASE:=1
 
 PKG_SOURCE:=go$(PKG_VERSION).src.tar.gz
-PKG_SOURCE_URL:=https://golang.org/dl/
-PKG_HASH:=f3de49289405fda5fd1483a8fe6bd2fa5469e005fd567df64485c4fa000c7f24
+PKG_SOURCE_URL:=https://golang.org/dl/ \
```

Comment:
This URL redirects to dl.google.com

Reply:
I believe I got this URL from the [Alpine Golang Docker image](https://github.com/docker-library/golang/blob/43f9c69e1bcb068bdb6883cd0fb2339e9b197510/Dockerfile-alpine.template#L42), when I was first trying to get gccgo working for OpenWrt. It appears the official downloads were hosted on golang.org for [1.3 and earlier](https://web.archive.org/web/20140812011700/http://golang.org:80/dl/), then they switched to storage.googleapis.com, and now dl.google.com.

I would prefer to keep the URL pointed to golang.org, as that is the official domain, and would presumably be future-proof if/when Google decides to change the download domain again.

(I also included dl.google.com in the list because the download links on [golang.google.cn](https://golang.google.cn/dl/), which is suppose to be accessible from mainland China, point to dl.google.com.)